packages/components/src/Components/TagCount/tagCount.scss

```diff
@@ -1,17 +1,17 @@
 .ins-c-tag-count {
     color: var(--pf-global--icon--Color--light);
     display: flex;
-    border: none;
-    padding: 0;
-    background-color: inherit;
 
-    i {
-        padding: 10px;
+    svg {
+        width: 30px;
+        height: 30px;
+        padding: 5px;
         font-size: var(--pf-global--icon--FontSize--lg);
     }
-
     span {
-        margin: auto 0;
+        position: absolute;
```

Comment:
We probably shouldn't use a position attribute here

Reply:
That's how I was able to center. If we're cool with the tag counts looking like this we're good
![tagCountComponentsNewUncentered](https://user-images.githubusercontent.com/26419550/65701357-29cf4b00-e04f-11e9-94aa-b0008f52f0e7.png)

Reply:
Can you flex the container and then `align-content: center`?